api/tasks.go

```diff
@@ -1167,3 +1169,9 @@ type WorkloadIdentity struct {
 	ServiceName string        `hcl:"service_name,optional"`
 	TTL         time.Duration `mapstructure:"ttl" hcl:"ttl,optional"`
 }
+
+type Action struct {
+	Name    string   `hcl:"name,label"`
+	Command string  `mapstructure:"command" hcl:"command"`
```

Comment:
This misalignment should be picked up by `gofmt` but weirdly it wasn't detected by our checks:

```suggestion
	Command string   `mapstructure:"command" hcl:"command"`
```